server/genquery2/src/genquery2_sql.cpp

```diff
@@ -428,50 +440,35 @@ namespace
         if (!_conditions.empty()) {
             sql += fmt::format(" where {}", _conditions);
 
-            if (_opts.admin_mode) {
-                if (d_iter != end && c_iter != end) {
-                    sql += fmt::format(" and pdoa.access_type_id >= {perm} and pcoa.access_type_id >= {perm}",
-                                       fmt::arg("perm", min_perm_level));
-                }
-                else if (d_iter != end) {
-                    sql += fmt::format(" and pdoa.access_type_id >= {}", min_perm_level);
-                }
-                else if (c_iter != end) {
-                    sql += fmt::format(" and pcoa.access_type_id >= {}", min_perm_level);
-                }
-            }
-            else {
+            if (!_opts.admin_mode) {
                 if (d_iter != end && c_iter != end) {
                     sql += fmt::format(" and pdu.user_name = ?"
                                        " and pdu.zone_name = '{zone}'"
                                        " and pdu.user_type_name != 'rodsgroup'"
-                                       " and pdoa.access_type_id >= {perm}"
+                                       " and pdoa.access_type_id >= 1050"
```

Comment:
why going to a 'magic' number here?   can we not use a variable for futureproofing / documentation?

Reply:
makes sense.    just want to make sure it's easy for the next reader to understand.